src/transformers/trainer.py

```diff
@@ -1997,14 +1997,23 @@ def _load_from_checkpoint(self, resume_from_checkpoint, model=None):
             model = self.model
 
         config_file = os.path.join(resume_from_checkpoint, CONFIG_NAME)
-
+        adapter_weights_file = os.path.join(resume_from_checkpoint, ADAPTER_WEIGHTS_NAME)
+        adapter_safe_weights_file = os.path.join(resume_from_checkpoint, ADAPTER_SAFE_WEIGHTS_NAME)
```

Comment:
Just for my own understanding, why don't we need the equivalent `adapter_weights_index_file` or `adapter_safe_weights_index_file` here?

Reply:
Sure! I think index files are used for sharded checkpoints only. In PEFT the saved checkpoints are always extremely light (order of magnitude of few MBs) - even for very large models - thus we never save sharded checkpoint, therefore we don't save index files!

Reply:
Thanks for explaining!